Azure.MgmtSdk.Analyzers.Test/IntervalPropertyNameAnalyzerTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+using VerifyCS = AzureMgmtSDKAnalyzer.Test.CSharpAnalyzerVerifier<
+    Azure.MgmtSdk.Analyzers.IntervalPropertyNameAnalyzer>;
+
+namespace Azure.MgmtSdk.Analyzers.Test
+{
+    [TestClass]
+    public class IntervalPropertyNameAnalyzerTests
+    {
+        [TestMethod]
+        public async Task AZM0020PublicBoolTypeWithVerbPrefix()
+        {
+            var test = @"namespace Azure.ResourceManager.Network
+{
+    public partial class Test
+    {
+        public int TestIntervalInSeconds { get; set; }
+        public int TestDurationInSeconds { get; set; }
+        public int Foo { get; set; }
+        public bool TestInterval { get; set; }
+        public int IntervalInSeconds { get; set; }
+        public int IntervalInMinutes { get; set; }
+    }
+}";
+            await VerifyCS.VerifyAnalyzerAsync(test); // Default No errors.
+        }
+
+        [TestMethod]
+        public async Task AZM0020PublicBoolTypeWithoutVerbPrefix()
+        {
+            var test = @"namespace Azure.ResourceManager.Network
+{
+    public partial class Test
+    {
+        public int FipInterval { get; set; }
```

Comment:
I recommend add some variables of other types.
Like 
+ public string TestInterval { get; set; }
+ public UInt64 TestDuration { get; set; }